ertviz/controllers/ensemble_selector_controller.py

```diff
@@ -78,11 +78,7 @@ def update_ensemble_selector(query, selected_ensembles, elements):
             ensemble_dict = get_ensembles(project_id=parent.project_identifier)
             for ensemble_schema in ensemble_dict:
                 ensemble_id = ensemble_schema["id"]
-                if ensemble_id not in parent.ensembles:
-                    parent.ensembles[ensemble_id] = EnsembleModel(
-                        ensemble_id=int(ensemble_id),
-                        project_id=parent.project_identifier,
-                    )
+                get_ensemble(parent, ensemble_id)
```

Comment:
Shouldn't this provide a handle to ensemble?

Reply:
maybe the name is wrong, as it not only returns the ensemble, but loads it...

Reply:
renamed function to load ensemble :)